spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -151,152 +155,139 @@ Explanation of the BNF elements:
 
 === Query by Method Name Keywords
 
-The following tables list the _Query by Method Name_ keywords that must be supported by Jakarta Data providers, except where explicitly indicated for a type of database.
+An implementation of Query by Method Name must support the following types of operation.
 
-[cols="12,~,25"]
+.Query by Method Name Actions
+[cols="12,~"]
 |===
-| Action | Description | Not Required For
+| Action | Description
 
 |`find`
 |Returns entity instances representing the records which satisfy the restriction, or representing all records if there is no restriction.
-|Key-value, Wide-Column
 
 |`delete`
 |Deletes every record which satisfy the restriction, or all records if there is no restriction, and returns either no result (`void`) or the number of records deleted.
-|Key-value, Wide-Column
 
 |`count`
 |Returns the number of records which satisfy the restriction, or the total number of records if there is no restriction.
-|Key-value, Wide-Column
 
 |`exists`
 |Returns `true` if at least one record satisfies the restriction or if there is at least one record in the database when there is no restriction.
-|Key-value, Wide-Column
 |===
 
-[NOTE]
-====
-The "Not Required For" column indicates the database types for which the respective keyword is not required or applicable.
-====
-Jakarta Data implementations must support the following list of Query by Method Name keywords, except where indicated for a database type. A repository method must raise `java.lang.UnsupportedOperationException` or a more specific subclass of the exception if the database does not provide the requested functionality.
+An implementation of Query by Method Name must support the following keywords.
 
-[cols="15,~,~,25"]
+.Query by Method Name Keywords
+[cols="15,~,~"]
 |===
-| Keyword |Description | Method signature Sample | Not Required For
+| Keyword | Description | Method signature example
```

Comment:
I don't agree with getting rid of the place to document the NoSQL imitations.

Reply:
I think this a good change. There was only one kind of limitation here, which is much more compactly written in a single sentence than taking up that extra space in the table.

Reply:
We have already informed that the database that does not support it will throw an UnsupportedOperationException.

Reply:
> I think this a good change. There was only one kind of limitation here, which is much more compactly written in a single sentence than taking up that extra space in the table.

I think the compactness is an inferior way to convey the information to the user that will be more difficult for them to understand, even more so as more restrictions are identified with NoSQL database types that need to be documented.